benchmark/src/main/scala/fs2/benchmark/RangeFoldBenchmark.scala

```diff
@@ -0,0 +1,14 @@
+package fs2
+package benchmark
+
+import org.openjdk.jmh.annotations.{Benchmark, Param, Scope, State}
+
+@State(Scope.Thread)
+class RangeFoldBenchmark {
+  @Param(Array("1", "4096", "665536", "67108863"))
+  var n: Int = _
+
+  @Benchmark
+  def rangeFold(): Option[Long] =
```

Comment:
General style question, is it better to add another `@Param` to the existing benchmark that is on `Stream` which organizes benchmarks by type, or separate it out like this>

Reply:
When adding an additional param, JMH will run the cross product -- e.g., n = 1, 2, 3 and m = 4, 5, 6 will cause JMH to run 9 cases (1, 4), (1, 5), (1, 6), (2, 4), ...

Reply:
Ahh, boooo